packages/cmf/src/bootstrap.md

```diff
@@ -0,0 +1,40 @@
+# react-cmf bootstrap
+
+This api help you to start without importing much than the api
+
+Quick start:
+
+```javascript
+import { api } from '@talend/react-cmf';
+import components from './components';
+import sagas from './sagas';
+
+configure.initialize();
+const app = api.bootstrap({
+    components,
+	settingsURL: '/settings.json',
+	saga: sagas.appSaga, // function* appSaga() {...}
+	sagaRouterConfig: sagas.routes, // { '/foo/bar': function* bar() {...}, ... },
+    ...manyMoreOptions
+});
+app.render();
+```
+
+## Options
+
+| name | type | default | description |
+| -- | -- | -- | -- |
+| settingsURL* | string | undefined | **REQUIRED** this URL to fetch the cmf settings.json file |
+| appId | string | 'app' | the default html element id to get to render react |
+| history | Object | `{ hashHistory } from 'react-router'` | the history object to control the router |
+| components | Object | undefined | an object with key/value to register your components |
+| actionCreators | Object | undefined | an object with key/value to register your action creators |
+| expressions | Object | undefined | an object with key/value to register your expressions |
+| sagas | Object | undefined | an object with key/value to register your sagas |
```

Comment:
Sorry, perhaps it's still me but having `an object with key/value to do something` makes it not as clear for the 4 entries above.
What do you think about that :

`A component dictionary where each key/value is registered in cmf store.`
```
function MyHeaderComponent() {}
function MyAsideComponent() {}
function MyMain() {}
const components = { MyHeaderComponent, MyAsideComponent, main: MyMain };
```

The examples are not really necessary for all the 4 entries, but it can be good to have a link to the entity spec such as expressions.